### PR TITLE
Update 03-vmware.md

### PR DESCRIPTION
**Pull request by antongisli**
pcg needs 60 GB

### CODE REVIEW

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1055#" title="Author self-approved">antongisli</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1055#pullrequestreview-1223590630" title="Approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/02-data-center/03-vmware.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/02-data-center/03-vmware.md:284]:
```suggestion
  + One (1) node - two (2) vCPU, 4 GB memory, 60 GB storage.
```

**Pull request merged**
